src/commands/aksCreateCluster/aksCreateCluster.ts

```diff
@@ -33,7 +33,7 @@ export default async function aksCreateCluster(_context: IActionContext, target:
                 vscode.window.showErrorMessage(subscriptionResult.error);
                 return;
             }
-            subscriptionId = subscriptionResult.result?.id;
+            subscriptionId = subscriptionResult.result?.subscriptionId;
```

Comment:
💡 note to self: double checking this object. Especially the relevance of `subscriptionResult.result?.id` vs subscriptionResult.result?. subscriptionId` and context it is used.

Reply:
subscriptionId is correct I had it wrong earlier in my previous PR. Kevin corrected it.